src/test/java/com/webaut/project/steps/TeamSteps.java

```diff
@@ -3,27 +3,35 @@
 import com.webaut.project.pages.HeaderCreateMenu;
 import com.webaut.project.pages.Home;
 import com.webaut.project.pages.Header;
+import com.webaut.project.pages.team.TeamDeleteConfirmation;
 import com.webaut.project.pages.team.TeamDetails;
 import com.webaut.project.pages.team.TeamForm;
+import com.webaut.project.pages.team.TeamSideBarOption;
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 import org.testng.Assert;
 
+import static java.lang.Thread.sleep;
+
 public class TeamSteps {
     private Home home;
     private TeamForm teamForm;
     private TeamDetails teamDetails;
     private Header header;
     private HeaderCreateMenu headerCreateMenuHeader;
+    private TeamSideBarOption teamSideBarOption;
+    private TeamDeleteConfirmation teamDeleteConfirmation;
 
-    public TeamSteps (Home home, TeamForm teamForm, TeamDetails teamDetails, Header header, HeaderCreateMenu headerCreateMenuHeader){
+    public TeamSteps (Home home, TeamForm teamForm, TeamDetails teamDetails, Header header, HeaderCreateMenu headerCreateMenuHeader, TeamSideBarOption teamSideBarOption, TeamDeleteConfirmation teamDeleteConfirmation){
```

Comment:
This code line is too long, maybe you should shorten it

Reply:
It is the price of Injection, I've talked Carlos About, and there is no problem, if you don´t have any another concern please, Merge it.

Reply:
Did you run gradle clean check ?

Reply:
the gradle clean check is not enabled on this project.
please skip that in the meanwhile.

Reply:
In my case, this command is working fine.

Reply:
JIC enabled Travis and fix all TC failed
please verify Travis is a success before merge this